build-container/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM archlinux:base-devel
+
+# The following is from https://github.com/archzfs/archzfs-ci/blob/master/worker/Dockerfile
+RUN pacman -Syu --noconfirm --needed python-pipx python-twisted python-future git wget systemd-sysvcompat openresolv vi
+
+# add buildbot user and give passwordless sudo access (needed for archzfs build scripts)
+RUN groupadd -r buildbot && \
+    useradd -m -g buildbot buildbot && \
+    mkdir /worker && \
+    chown buildbot:buildbot /worker && \
+    echo "buildbot ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+
+# aur prep
+RUN useradd aur && \
+    echo "aur ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+
+# clean-chroot-manager
+USER aur
+RUN mkdir -p /tmp/ccm-install            && \
+    cd /tmp/ccm-install                  && \
+    wget https://aur.archlinux.org/cgit/aur.git/snapshot/clean-chroot-manager.tar.gz && \
```

Comment:
We could use `curl` here, especially if we get rid of `wget` at some point, but it's okay for now IMO. I suppose we could've used `git` too, but the simpler, the better, so fetching the tar.gz is a good call I think.